HW02_A_Graph-Fails.md

```diff
@@ -0,0 +1,189 @@
+
+``` r
+ggplot(data = mpg) +
+  geom_point(mapping = aes(x = cty, y = hwy, color = "blue")) 
+```
+
+![](HW02_A_Graph-Fails_files/figure-gfm/unnamed-chunk-2-1.png)<!-- -->
+
+*Graph 1 Solution \#2*
+
+``` r
+ggplot(mpg, aes(x = cty, y = hwy, color = "blue")) + geom_point()
+```
+
+![](HW02_A_Graph-Fails_files/figure-gfm/unnamed-chunk-3-1.png)<!-- -->
+
+### Graph Fail 2
+
+``` r
+ggplot(data = mpg) + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = "blue"))
+```
+
+![](HW02_A_Graph-Fails_files/figure-gfm/unnamed-chunk-4-1.png)<!-- -->
+
+**Graph 2 Answer:** If you want to change the color of the points, the
+“color” argument should be outside the aes function. You assign
+variables to “color” within the aes function if you want to change the
+color of the graph according to levels of that variable.
+
+*Graph 2 Solution*
+
+``` r
+ggplot(data = mpg) + 
+  geom_point(mapping = aes(x = displ, y = hwy), color = "blue")
+```
+
+![](HW02_A_Graph-Fails_files/figure-gfm/unnamed-chunk-5-1.png)<!-- -->
+
+### Graph Fail 3
+
+``` r
+mpg %>% 
+ggplot() + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = class), alpha = 2) + 
+  theme(legend.direction = "horizontal") + 
+  theme(legend.position = c(5, 40))
+```
+
+![](HW02_A_Graph-Fails_files/figure-gfm/unnamed-chunk-6-1.png)<!-- -->
+
+**Graph 3 Answer:** Alpha changes the opacity. Setting alpha = 2 doesn’t
+do anything because 100% opacity is already reached at alpha = 1. To
+increase point size, use the argument “size”. Furthermore, to move the
+legend, use the theme function and the legend.position() argument.
+c(x,y) cannot be greater than (1,1) because (1,1) specifies the top
+right location of the graph. To remove elements from the graph, set
+“element\_blank()” in theme() elements; see below. To remove the
+legend entirely, add layer: theme(legend.position = “none”)
+
+*Graph 3 Solution*
+
+``` r
+mpg %>% 
+ggplot() + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = class), size = 2) + 
+  theme(legend.direction = "horizontal") + 
+  theme(legend.position = c(0.6, .8), legend.title=element_blank()) 
+```
+
+![](HW02_A_Graph-Fails_files/figure-gfm/unnamed-chunk-7-1.png)<!-- -->
+
+``` r
+  # + theme(legend.position = "none") #to remove legend
+```
+
+### Graph Fail 4
+
+``` r
+mpg %>% 
+ggplot(mapping = aes(x = displ, y = hwy, color = drv)) + 
+  geom_point() + 
+  geom_smooth(se = F) #se = F makes it so it won't show the error in the line of fit
+```
+
+    ## `geom_smooth()` using method = 'loess' and formula 'y ~ x'
+
+![](HW02_A_Graph-Fails_files/figure-gfm/unnamed-chunk-8-1.png)<!-- -->
+
+**Graph 4 Answer:** Geom\_smooth is creating a line for each level of
+“drv” because the data has been grouped by drv levels. If just one
+line is wanted for all the data points, remove the color argument.
+
+*Graph 4 Solution*
+
+``` r
+mpg %>% 
+ggplot(mapping = aes(x = displ, y = hwy)) + 
+  geom_point() + 
+  geom_smooth(se = F) 
```

Comment:
I see you already saw my version of this that maintains the color aesthetic. I wonder what you would do if you wanted to make the overall trend line and the category specific lines? Maybe you would make two geom_smooth layers?